openwisp_users/api/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.conf.urls import url
+from openwisp_users import settings as app_settings
+
+from . import views
+
+urlpatterns = []
+
```

Comment:
I gave it some more thought and concluded it will be better if we namespace the URLs (as we do in other modules), can we add `app_name = 'users'` to divide these endpoints in this namespace please?